app/editor/src/features/admin/ministers/MinisterList.tsx

```diff
@@ -18,7 +18,6 @@ export const MinisterList: React.FC = () => {
   const [items, setItems] = React.useState<IMinisterModel[]>([]);
 
   React.useEffect(() => {
-    console.log('a thing!');
     if (!ministers.length) {
```

Comment:
If the API doesn't return any it can result in an infinite loop

Reply:
This work wasn't actually from me, just did a search for `console.log` and found another one left in error, I will create a ticket to capture this work